packages/local-signing-manager/src/lib/local-signing-manager.ts

```diff
@@ -101,10 +101,10 @@ export class LocalSigningManager implements SigningManager {
    *
    * @param args.accounts - array of private keys
    */
-  public static async create(args: { accounts: PrivateKey[] }): Promise<LocalSigningManager> {
+  public static async create(args: { accounts: PrivateKey[], curveType?: Curve }): Promise<LocalSigningManager> {
     await cryptoWaitReady();
 
-    return new LocalSigningManager(args.accounts);
+    return new LocalSigningManager({ accounts: args.accounts, curveType: args.curveType } = { accounts:[], curveType: 'sr25519' });
```

Comment:
This doesn't look right to me. Is this inline assignment, and then passing it in?

The `= { ... }` part needs to be in definition of the function, and not at the call site.